### PR TITLE
Create news item rich text

### PR DESCRIPTION
**Pull request by nvpmai95**
- Save and display rich text in html format
- Integrate Quilljs into Rails admin's field body of News. Save successfully

### CODE REVIEW

File: app/views/news_items/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<%= render 'flash_messages' %>
+<!-- Include Quill stylesheet -->
+<% content_for :head do %>
+  <link href="https://cdn.quilljs.com/1.0.6/quill.snow.css" rel="stylesheet">
+<% end %>
+
+<%= form_for NewsItem.new, url: {action: "create"} do |f| %>
```
Comment (harley) [inline app/views/news_items/new.html.erb]: `<%= form_for NewsItem.new do |f| %>` should be enough. is there an edit/update form?

Comment (harley) [inline app/views/news_items/new.html.erb]: do people only edit in rails_admin?

Comment (nvpmai95) [inline app/views/news_items/new.html.erb]: If we can use the editor in Rails admin form, we won't need this anymore. I should have deleted it. 
The partial for our editor is [here](https://github.com/blueskyacademy/new-bluesky/blob/create_news_item_rich_text/app/views/rails_admin/main/_quilljs_form.html.erb)

**Comment by harley** [comment]
thanks 👍

**Pull request merged**